google-cloud-datastore/src/main/java/com/google/cloud/datastore/testing/LocalDatastoreHelper.java

```diff
@@ -320,6 +344,8 @@ public void reset() throws IOException {
    */
   @Override
   public void stop(Duration timeout) throws IOException, InterruptedException, TimeoutException {
+    // TODO(gapic_upgrade): Temporarily addressing the flaky connection refused error
```

Comment:
To avoid any potential tech debt, can we move this to src/test instead?

Reply:
You mean moving LocalDatastoreHelper to src/test ?

Reply:
no sorry, just the `checkHealth` call

Reply:
so due to this two tests were becoming flaky, DatastoreTest (in same module) and ConceptsTest (in sample module). moving it to src/test will fix the case of the current module but not the sample one as it won't be accessible without having some additional configurations in maven. 

How about we make it package private, so it remains an internal implementation for us ?

Reply:
I think it's fine to come back to the ConceptsTest as a todo - since this is really for test code, I'd prefer it in the test module. May also be worth filing an issue as to why it's flaky in the first place.

Reply:
moved to src/test in 7f4ce8d.